README.md

```diff
@@ -8,75 +8,102 @@ We more than welcome contributions to KubeVirt documentation. Please reach out i
- `git commit -s -m "The commit message" file1 file 2 ...`
+```console
+$ export CONTAINER_ENGINE=docker
+$ export BUILD_ENGINE=docker
+$ make run
+```
 
-- Create GitHub pull request to master branch
+Open your web browser to http://0.0.0.0:8000 and validate page rendering
+
+6. Commit your code after performing signature verification
+
+Signature verification is required -- you may sign your commits by running:
 
-#### Make Help
 ```console
+$ git commit -s -m "The commit message" file1 file 2 ...
+```
+
+7. Create GitHub pull request to master branch
+
+#### Make Help
 
+```console
 Makefile for user-guide mkdocs application
 
 Usage:
   make <target>
 
 Env Variables:
-  CONTAINER_ENGINE	Set container engine, [*podman*, docker]
-  BUILD_ENGINE		Set build engine, [*podman*, buildah, docker]
-  SELINUX_ENABLED	Enable SELinux on containers, [*False*, True]
+  CONTAINER_ENGINE      Set container engine, [*podman*, docker]
+  BUILD_ENGINE          Set build engine, [*podman*, buildah, docker]
+  SELINUX_ENABLED       Enable SELinux on containers, [*False*, True]
+  PYTHON                Python executable, [*python3.7*, python]
+  PIP                   Pip executable, [*pip3*, pip]
+  LOCAL_SERVER_PORT     Port on which the local mkdocs server will run, [*8000*]
 
 Targets:
-  help                	 Show help
-  check_links         	 Check external and internal links
-  check_spelling      	 Check spelling on site content
-  build_image_userguide	 Build image: userguide
-  build_image_yaspeller	 Build image: yaspeller
-  build               	 Build site. This target should only be used by Prow jobs.
-  run                 	 Run site.  App available @ http://0.0.0.0:8000
-  status              	 Container status
-  stop                	 Stop site
-  stop_yaspeller      	 Stop yaspeller image
+  help                   Show help
+  check_links            Check external and internal links
+  check_spelling         Check spelling on site content
+  build_image_userguide  Build image: userguide
+  build_image_yaspeller  Build image: yaspeller
+  build                  Build site. This target should only be used by Prow jobs.
+  run                    Run site.  App available @ http://0.0.0.0:8000
+  status                 Container status
+  stop                   Stop site
+  stop_yaspeller         Stop yaspeller image
 ```
 
 #### Environment Variables
-* CONTAINER_ENGINE: Some of us use docker. Some of us use podman.
+* `CONTAINER_ENGINE`: Some of us use `docker`. Some of us use `podman` (default: `podman`).
+
+* `BUILD_ENGINE`: Some of us use `docker`. Some of us use `podman` or `buildah` (default: `podman`).
+
+* `SELINUX_ENABLED`: Some of us run SELinux enabled. Set to `True` to enable container mount labelling.
+
+* `PYTHON`: Change the `python` executable used (default: `python3.7`).
 
-* BUILD_ENGINE: Some of us use docker. Some of us use podman or buildah.
+* `PIP`: Change the `pip` executable used (default: `pip3`).
```

Comment:
We have to hide this.  Too much chance for problems with wrong modules getting installed.

Reply:
Done, I'm fine with taking this out entirely -- 99% of the time looks like people won't be changing this anyway

- [ ] Remove `PIP` note from usage and README to avoid people changing it